compiler/ilgen/OMRVirtualMachineRegisterInStruct.hpp

```diff
@@ -67,7 +67,11 @@ class VirtualMachineRegisterInStruct : public TR::VirtualMachineRegister
       _fieldName(fieldName),
       _localNameHoldingStructAddress(localNameHoldingStructAddress)
       {
-      _elementType = b->typeDictionary()->GetFieldType(structName, fieldName)->baseType()->baseType();
+      _elementType = b->typeDictionary()->GetFieldType(structName, fieldName)->baseType();
```

Comment:
I find this code confusing because _elementType is named for what we store there, but it's not well named for the purpose for which we use it (to cook up integer values when we want to adjust the register's value). Confusing because the _elementType is not Word even if the field's base type is a pointer type. But it's a perfectly reasonable thing to do once you look at how _elementType is used in OMR::VirtualMachineRegister.
 
I'd like to propose that we also change the name of _elementType (in OMR::VirtualMachineRegister) to something like _integerTypeForAdjustments, which can be initialized as _elementType is being initialized below. At that point, I don't think I'll find this code confusing at all, which will at least make me happier.

Can I ask you to update the pull request with that change, please, @charliegracie ?